tests/polkadotjs_test/test/test-polkadot.js

```diff
@@ -11,7 +11,7 @@ describe('Testing polkadot.js/api calls:', function () {
     let done = false;
 
     before (async function () {
-        const wsProvider = new WsProvider('ws://127.0.0.1:8546');
+        const wsProvider = new WsProvider(process.env.WSHOST);
```

Comment:
Interesting. Maybe you could copy the js files to a `t.TempDir`, modify them there. That way you can run each test in parallel without depending on the shared env variable or shared files